test/groovy/templates/PiperPipelineStageInitTest.groovy

```diff
@@ -128,6 +133,24 @@ class PiperPipelineStageInitTest extends BasePiperTest {
 
     }
 
+    @Test
+    void testCustomStashSettings() {
+        jryr.registerYaml('customStashSettings.yml',"unstash: source")
```

Comment:
Could you make it a bit more realistic maybe? This would not work as stash settings go, right?

Reply:
I mean, this is not to test that our changed stash settings actually do anything different, but just that we can set the value. I think having it much longer to mimic the normal settings will only add confusion/bulk..